### DISCUSSION TITLE
After recovery, channels which appear online are not sweeping/closing down

### DISCUSSION THREAD

**Discussion by dunkfan** [lightningnetwork/lnd#8050] [Troubleshooting]
I have a couple of channels which are not closing down after recovery with an SCB was started.
They are listed in waiting_close_channels.
The nodes of the channels appear to be online!
Is there any point in  running restorechanbackup a second time?

**Comment by guggero** [comment]
If the peer is definitely online (check with `lncli listpeers`), then they might be a certain version of CLN that didn't respond to the recovery request. You can try [`chantools triggerforceclose`](https://github.com/lightninglabs/chantools/blob/master/doc/chantools_triggerforceclose.md). And sometimes restarting the node to re-trigger the SCB process also helps.
If that doesn't work, you can always try to contact the peers and ask them to force close from their end.

**Reply by dunkfan** [reply]
One of them is in the listpeers list, both show a red flag in Amboss "Last Update" 59 and 188 hours since the last update.
Why does triggerforceclose require the rootkey obtained from the original lncli createwallet showrootkey, proof of identity I guess?
channel_point is available from the pendingchannels list? Thanks again.

**Reply by guggero** [reply]
Yes, the p2p communication is encrypted and needs access to the node identity's private key (which is derived from the seed).
Yes, the channel point is in the pending channels list.
